File: Java/UberVan.java
// import java.util.ArrayList;
// import java.util.Map;

class UberVan extends CarAdvanced {  
  
  // public UberVan(String license, Account driver, Map<String, Map<String,Integer>> typeCarAccepted, ArrayList<String> seatsMaterial) {
  //   super(license, driver, typeCarAccepted, seatsMaterial);
  // }

  public UberVan(String license, Account driver) {
    super(license, driver);
  }
    
  @Override  
  public void setPassengers(Integer passengers) {
    if (passengers == 6) {
      this.passengers = passengers;
    } else {
      System.out.println("Necesitas asignar 6 pasajeros");
    }
  }
}

File: PHP/UberVan.php
<?php 
  require_once('CarAdvanced.php'); 
  class UberVan extends CarAdvanced {
    public function __construct($license, $driver, $typeCarAccepted, $SeatsMaterial) {
      parent::__construct($license, $driver, $typeCarAccepted, $SeatsMaterial);
    }

    public function setPassengers($passengers){
      if ($passengers == 6) {
        $this->passengers = $passengers;        
      } else {
        echo "Necesitas asignar 6 pasajeros";
      }
    }

    public function printCarData(){
      echo "
        License: $this->license 
        Conductor: {$this->driver->name}  
        Pasajeross: $this->passengers 
      ";
    }
  }
?>